### PR TITLE
Us133f borrower confirms deed

### PR DESCRIPTION
**Pull request by flynnbops**
Acceptance tests to check deed status is updated throughout signing process

### CODE REVIEW

File: features/lr_casework_search_for_deed.feature [hunk 1]
```diff
@@ -32,11 +32,54 @@ And I create the deed via the Deed API
...
+# "DRAFT"
+# "PARTIALLY-SIGNED"
+# "SUPERCEDED"
+# "ABORTED"
+# "STOPPED"
+# "ALL-SIGNED"
+# "EFFECTIVE"
+# "SUBMITTED"
+# "STORED"
+# "REGISTERED"
```
Comment (nigelpain) [inline features/lr_casework_search_for_deed.feature]: Not sure if half way through feature file is best place to store this information, it breaks up the document and I'm not sure if it really adds anything here, feels like something that should be in deed-api readme.md

File: features/step_definitions/lr_casework_search_for_deed.rb [hunk 2]
```diff
@@ -6,16 +6,17 @@
   @response = HTTP.get(full_url)
 end
 
-Then(/^I verify the returned deed information$/) do
+Then(/^I verify the returned deed information(?: is "([^"]*)")?$/) do |deed_sts|
```
Comment (nigelpain) [inline features/step_definitions/lr_casework_search_for_deed.rb]: status seems like a more descriptive vairable name than deed_sts

File: features/step_definitions/lr_casework_search_for_deed.rb [hunk 3]
```diff
@@ -6,16 +6,17 @@
...
     deed_count = 0
     # Checks each returned deed for key values: status, token. And value DRAFT.
     while deed_count <= (data.length - 1)
-      status = data[deed_count].include?('status')
-      token = data[deed_count].include?('token')
-      draft = data[deed_count]['status'].include?('DRAFT')
-      tok_val = data[deed_count]['token'].nil?
+      status = data[deed_count.to_i].include?('status')
+      token = data[deed_count.to_i].include?('token')
+      draft = data[deed_count.to_i]['status'].include?(deed_sts)
```
Comment (nigelpain) [inline features/step_definitions/lr_casework_search_for_deed.rb]: Would be tempted to rename this from draft to soemthing more meaningful now status is a variable thing, something like status_val or whatever you think is more suitable.

File: features/lr_casework_search_for_deed.feature (hunk 1 above)
Comment (flynnbops) [inline features/lr_casework_search_for_deed.feature]: I removed this from the Feature

File: features/step_definitions/lr_casework_search_for_deed.rb (hunk 2 above)
Comment (aaronFlynn) [inline features/step_definitions/lr_casework_search_for_deed.rb]: Updated variable name

File: features/lr_casework_search_for_deed.feature [hunk 4]
```diff
@@ -32,11 +32,54 @@ And I create the deed via the Deed API
...
+And I search the deed store with title_number "99999"
+Then I verify the returned deed information is "DRAFT"
+And borrower <1> views the deed
+And the borrower <1> signature element is present on page
+Then the deed is digitally signed by borrower <1>
+When I search the deed store with title_number "99999"
+Then I verify the returned deed information is "ALL-SIGNED"
+
+@us133f @us152
+Scenario: Verify deed status is PARTIALLY-SIGNED when not fully signed
```
Comment (nigelpain) [inline features/lr_casework_search_for_deed.feature]: Feel like these scenarios could do with a rename just to make it clearer what partially signed and fully signed is. So maybe something like 'Verify deed status is PARTIALLY SIGNED when only one of two borrowers have signed the deed/all borrowers have signed the deed' or something like that perhaps.

**Comment by nigelpain** [comment]
Few minor comments around naming but happy to :+1: this in general it looks good.

**Comment by flynnbops** [comment]
CLOSING: Us133f borrower confirms deed V2 has been merged instead

**Pull request closed without merging**